io-ballerina/src/io/tests/unit-tests/bytes_io.bal

```diff
@@ -73,6 +72,33 @@ function testWriteBytes() {
     closeWritableBytesChannel();
 }
 
+@test:Config {}
+function testFileWriteBytes() {
+    string filePath = TEMP_DIR + "bytesFile2.txt";
+    createDirectoryExtern(TEMP_DIR);
+    string content = "Sheldon Cooper";
+    var result = fileWriteBytes(filePath, content.toBytes());
+    
+    if (result is Error) {
+        test:assertFail(msg = result.message());
+    }
+}
+
+@test:Config {
+    dependsOn: ["testFileWriteBytes"]
```

Comment:
IMO, `testFileReadBytes` doesn't have to depend on `testFileWriteBytes` unless testFileReadBytes` needs to read bytes from the file generated by testFileWriteBytes` previously. If that's the case isn't it better to read from a resource file and remove the dependency on the other test.